tests/visualization_tests/test_slice.py

```diff
@@ -374,10 +379,10 @@ def test_get_slice_plot_info_nonfinite_multiobjective(objective: int, value: flo
         study,
         params=["param_b", "param_d"],
         target=lambda t: t.values[objective],
-        target_name="Objective Value",
+        target_name="Testing Nonfinite Multiobjective",
```

Comment:
Since `Target Name` is used in other tests for example https://github.com/optuna/optuna/blob/3247030717413b5097db10a26afca67322c91973/tests/visualization_tests/test_slice.py#L83, using the same value would be consistent, which improves maintainability:

```suggestion
        target_name="Target Name",
```